target/linux/lantiq/image/Makefile

```diff
@@ -24,6 +24,14 @@ define Build/append-avm-fakeroot
 	cat ./eva.dummy.squashfs >> $@
 endef
 
+define Build/append-uImage-dummyrootfs
+	echo -n "dummy" > $@.dummyrootfs
```

Comment:
What is that about? Looks like maybe embedding a rootfs into the kernel for debugging, but not production?

Reply:
I guess the bootloader just expects this to exist, and we put a dummy to make the bootloader happy.

Reply:
While my image can not do much yet, it boots just fine without. And i am almost sure i still have the original bootloader on the device.

Reply:
Good. So maybe this was just copy&pasted from another device which may need it and we don't need it for this one.